client/src/pages/homepage/HomePage.jsx

```diff
@@ -44,14 +44,47 @@ export default function HomePage() {
         {videoData.slice(0, 5).map((v) => (
           <SwiperSlide key={v.title} id="heroSlide">
             <NavLink to={`/video/${v.id}`}>
-              <img id="imageSlider" src={v.image} alt={v.title} />{" "}
+              <img id="imageHero" src={v.image} alt={v.title} />{" "}
```

Comment:
Using "Id" in any virtual dom based library/framework is bad practice because you are bypassing what the tools is made for. 

Replace this by a className or explore any other solutions over the Internet 😉